### PR TITLE
ArangoDB Implementation

### PR DESCRIPTION
**Pull request by robobenklein**
Running top1k right now, looking good so far.

Supports switching DBs naturally via CLI flags, so top100 / 1k/ 10k/ etc can each be managed easier.

Still learning the query language, might also get into graphQL.

### CODE REVIEW

**Comment by jball1997** [comment]
Do I trust the Travis Cl checks?

File: tests/test-python-arango-insertion.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        "-l", "--language",
+        type=str,
+        help="Language to parse",
+        required=True
```
Comment (solsane) [inline tests/test-python-arango-insertion.py:26]: 1 Lang at a time?

**Review by solsane — APPROVED** [review]

File: tests/test-python-arango-insertion.py (hunk 1 above)
Comment (robobenklein) [reply tests/test-python-arango-insertion.py:26]: Yes, this file is for evaluating node insert performance so it only accepts / parses one file.

**Comment by robobenklein** [comment]
This is looking like all systems ready for launch...

When seneca gets installed I will need to investigate how we want to store / deduplicate node text at scales greater than top1k.

**Pull request merged**